### ISSUE TITLE
Questions regarding the specification.

### ISSUE THREAD

**Issue by aasmune** [TTK4145/Project#4]
Section 3.3 in the specification states that two errors may not happen simultaneously, or within reasonable time for recovering from the first error.
Is it also safe to assume:
1. That two errors may not be resolved simultaneously, or within reasonable time?

2. When an error is resolved, another error will not happen within reasonable time.

**Comment by kjetilkjeka** [comment]
@aasmune 

> 1. That two errors may not be resolved simultaneously, or within reasonable time?

I don't think this is a valid assumption. At least all elevators might be started at the same time. So they might appear on the network almost simultaneously. I will need to think about it. Do you have an example of why this might be a problem?

> 2. When an error is resolved, another error will not happen within reasonable time.

I will need to think about it, please give an example of why this could be problematic.
